webservice/test/test_bagitutils.py

```diff
@@ -105,6 +108,25 @@ def test_get_suffix(self):
         self.assertTrue(suffix, '0')
         self.assertRaises(ValueError, bag.get_suffix, 'not_a_filetype')
 
+    def test_only_one_cram_in_data(self):
+        """The data we're loading here only contains one row, and that row
+        contains a CRAM file. We tests whether "samples" has a column name
+         "file_type2"."""
+        bag = BagHandler(data=self.data3, bag_info={}, bag_name='manifest')
+        participants, max_files_in_sample, protocols, file_types = \
+            bag.participants_and_max_files_in_sample_and_protocols()
+        self.assertEqual(len(participants), 1)
+        self.assertEqual(len(protocols), 1)
+        self.assertEqual(max_files_in_sample, 1)
+
+        samples = bag.samples(protocols, file_types)
+        self.assertEqual(len(samples[0].keys()), 9)
```

Comment:
Might be nice to instead/also assert that `file_type1` and `file_type0` don't exist, which is clearer than the magic number 9.